addons/hr_holidays/views/hr_leave_type_views.xml

```diff
@@ -81,7 +83,8 @@
                             <field name="show_on_dashboard"/>
                             <field name="support_document" string="Allow To Attach Supporting Document" />
                             <field name="time_type" required="1"/>
-                            <field name="company_id" groups="base.group_multi_company"/>
+                            <field name="company_id" groups="base.group_multi_company" readonly="has_been_used"/>
+                            <field name="country_id" groups="base.group_multi_company" readonly="company_id or has_been_used"/>
```

Comment:
Yes :) Basically the field `overtime_deductible` is above when `hr_holidays_attendance` is installed. As it's display will be depending on the fields you're adding, it will make the display "jump" under the user's mouse, which is to be avoided as much as possible :+1: